rad/src/script.rs

```diff
@@ -110,6 +114,56 @@ fn errorify(kind: RadError) -> RadError {
     kind
 }
 
+pub fn create_radon_script(
+    filters: &[RADFilter],
+    reducer: u32,
+) -> Result<Vec<RadonCall>, RadError> {
+    let unknown_filter = |code| RadError::UnknownFilter { code };
+    let unknown_reducer = |code| RadError::UnknownReducer { code };
+
+    let mut radoncall_vec = vec![];
+    for filter in filters {
+        let filter_op = filter.op;
```

Comment:
I see `filter_op as i128` many times, why not just do it here?

```suggestion
        let filter_op = filter.op as i128;
```